docs/docs/introduction/getting-started.md

```diff
@@ -0,0 +1,44 @@
+---
+id: getting-started
+title: Getting Started with electron-redux
+description: 'Introduction > Getting Started: Resources to get started with electron-redux'
+hide_title: false
+---
+
+Electron-Redux is an Redux Store Enhancer that helps you loosely synchronize the redux stores in multiple electron processes.
```

Comment:
```suggestion
electron-redux is a Redux Store Enhancer that helps you loosely synchronize the redux stores in multiple electron processes.
```

Reply:
Related - what do you mean by "loosely synchronise"?

Reply:
@hardchor Maybe the wording isn't right, but I want to emphasize the fact that the stores in main/renderer are not actually synchronized out of the box and there is no mechanism with the library that will actually provide that - we don't sync the states, we just make sure the actions are accessible from both places. Any clues on different more self-explanatory naming?